packages/application-shell/src/apollo-links/header-link.ts

```diff
@@ -16,35 +18,36 @@ import {
 
 declare let window: ApplicationWindow;
 
-type GraphQlTarget = typeof GRAPHQL_TARGETS[keyof typeof GRAPHQL_TARGETS];
-
 type ApolloContextWithInMemoryCache = TApolloContext & {
   cache: ApolloClient<NormalizedCacheObject>;
 };
 
 type QueryVariables = {
-  target: GraphQlTarget;
+  // Deprecated, use `{ context: { target } }`
+  target?: TGraphQLTargets;
```

Comment:
Those metadata options in the `variables` object are now deprecated (for backwards compatibility) and instead should be passed using the `context` object.